frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts

```diff
@@ -231,6 +232,9 @@ export const pipelineRunDetailsPO = {
     },
     status: '.odc-taskrun-details__status',
   },
+  parameters: {
+    form: '[data-test="pipelineRun-parameters"]',
```

Comment:
`form: '[data-test="pipeline-parameters"]'` instead of `form: '[data-test="pipelineRun-parameters"]'`

<img width="1562" alt="image" src="https://user-images.githubusercontent.com/65410551/179477789-e13c4d67-be1f-40dd-9444-a5bd4cb8047a.png">

Reply:
Forgot to update the `data-test` after the component refactoring. Updated it now. PTAL again.
<img width="625" alt="image" src="https://user-images.githubusercontent.com/2561818/179495616-9397fbd9-061e-41a2-b661-aa8b8c869705.png">